src/erlzk_conn.erl

```diff
@@ -396,7 +396,9 @@ connect(State=#state{servers = Servers}) ->
     case connect(ResolvedServers, rotate_server(State)) of
         {error, Retry} ->
             %% Maybe try again later
-            if Retry -> erlang:send_after(?ZK_RECONNECT_INTERVAL, self(), reconnect);
+            if Retry ->
+                    ReconnInterval = application:get_env(erlzk, zk_reconn_interval_ms, ?ZK_RECONNECT_INTERVAL),
```

Comment:
Nit: zk_reconnect_interval_ms

Reply:
FYI, this repo does not follow rule of 80 max chars per line at all.